### DISCUSSION TITLE
deploy in k8s.

### DISCUSSION THREAD

**Discussion by Goutami-Gowda** [apache/kvrocks#1175] [Q&A] [answered]
hello, has any one tried to run kvrocks on K8s?
Is there any experience to share?
 i deployed kvrocks cluster on k8s as daemonset but pod are not able to communicate with each other. so every pod is now  a standalone cluster.  i am followed this document.. but its not helping.
 https://kvrocks.apache.org/docs/Cluster/kvrocks-cluster-introduction/

**Comment by uds5501** [comment, marked as answer]
I did not pdo it on k8s but converting the docker compose to k8s deployment files should be easy enough, here's a small sample - https://github.com/apache/incubator-kvrocks/discussions/1485

Feel free to perform a POC !
